python/pyspark/pandas/frame.py

```diff
@@ -5266,12 +5266,12 @@ def to_orc(
         **options: "OptionalPrimitiveType",
     ) -> None:
         """
-        Write the DataFrame out as a ORC file or directory.
+        Write a DataFrame to the ORC format.
```

Comment:
FYI, It's changed just for matching the pandas document.
![Screen Shot 2022-09-27 at 5 18 04 PM](https://user-images.githubusercontent.com/44108233/192472375-c5ed5dd0-04a4-41fb-b7e8-0801c353cb59.png)